docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -433,38 +436,33 @@ on two data sets and different hardware configurations. Your data set has an
-|==============================================================================================================
-
-The `msmarco-long-light` data set contains long documents with an average of 
-over 512 tokens, which provides insights into the performance implications 
-of indexing and {infer} time for long documents. This is a subset of the 
-"msmarco" dataset specifically designed for document retrieval (it shouldn't be 
-confused with the "msmarco" dataset used for passage retrieval, which primarily 
-consists of shorter spans of text). 
-
-The `arguana` data set is a https://github.com/beir-cellar/beir[BEIR] data set. 
-It consists of long queries with an average of 200 tokens per query. It can 
-represent an upper limit for query slowness.
-
-The table below present benchmarking results for ELSER using various hardware 
-configurations.
-
-|==================================================================================================================================================================================
-|                                                         3+^| `msmarco-long-light`                                     3+^| `arguana`                                             | 
-|                                                         ^.^| inference     ^.^| indexing         ^.^| query latency   ^.^| inference      ^.^| indexing      ^.^| query latency  |  
-| **ML node 4GB - 2 vCPUs (1 allocation * 1 thread)**     ^.^| 581   ms/call ^.^| 1.7   doc/sec    ^.^| 713   ms/query  ^.^| 1200   ms/call ^.^| 0.8   doc/sec ^.^| 169   ms/query |  
-| **ML node 16GB - 8 vCPUs (7 allocation * 1 thread)**    ^.^| 568   ms/call ^.^| 12    doc/sec    ^.^| 689   ms/query  ^.^| 1280   ms/call ^.^| 5.4   doc/sec ^.^| 159   ms/query |  
-| **ML node 16GB - 8 vCPUs (1 allocation * 8 thread)**    ^.^| 102   ms/call ^.^| 9.7   doc/sec    ^.^| 164   ms/query  ^.^| 220    ms/call ^.^| 4.5   doc/sec ^.^| 40    ms/query | 
-| **ML node 32 GB - 16 vCPUs (15 allocation * 1 thread)** ^.^| 565   ms/call ^.^| 25.2  doc/sec    ^.^| 608   ms/query  ^.^| 1260   ms/call ^.^| 11.4  doc/sec ^.^| 138   ms/query | 
-|==================================================================================================================================================================================
+==== ELSER V2
+
+Overall the platform specific V2 model ingested at a max rate of 26 docs/s, 
+compared with the ELSER V1 max rate of 14 docs/s from the ELSER V1 benchamrk, 
+resulting in a 90% increase in throughput.
+
+The performance of virtual cores (that is, when the number of allocations is 
+greater than half of the vCPUs) has increased. Previously, the increase in 
+performance between 8 and 16 allocations was around 7%. It has increased to 17% 
+(ELSER V1 on 8.11) and 20% (for ELSER V2 platform specific). Tests were 
+performed on a 16vCPU machine.
```

Comment:
Lets add the number of tokens per document here:

"These tests were performed on a 16vCPU machine, with all documents containing exactly 256 tokens."

Reply:
We might want to add another sentence like "The length of the documents in your dataset will have a significant impact on your throughput numbers." or similar